specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/definitions.json

```diff
@@ -1382,6 +1382,16 @@
       },
       "description": "Authorization login response contract."
     },
+    "AuthorizationConfirmConsentCodeRequestContract": {
+      "type": "object",
+      "properties": {
+        "consentCode": {
```

Comment:
Does the consent code needs to be a secret? If yes, please x-ms-secret annotation

Reply:
No, it does not need to be a secret